modules_extra/notifications.js

```diff
@@ -106,15 +110,15 @@ exports.create = function (api) {
   }
 
   function getFirstMessage(feedId, cb) {
-    api.sbot_user_feed({id: feedId, gte: 0, limit: 1})(null, cb)
+    api.sbot.user_feed({id: feedId, gte: 0, limit: 1})(null, cb)
```

Comment:
I'm actually fine with either. I have found that using snake case makes the depject modules stand out loud and clear, which has been a god-send for moving and renaming things